web_m2x_options/static/src/js/form.js

```diff
@@ -21,6 +21,46 @@ odoo.define('web_m2x_options.web_m2x_options', function (require) {
         method: 'get_web_m2x_options',
     });
 
+    function is_option_set(option){
```

Comment:
```suggestion
    var is_option_set = function is_option_set(option){
```

Reply:
@TDu For my information, what is the benefit of declaring the function variable this way (or which style guide are we honoring)?

If I understand correctly, in your suggestion it is declared twice, because `function is_option_set(option) {})` already declares the variable by itself.

Reply:
Well looking at it again I should have suggested this
```suggestion
var is_option_set = function (option){
```
Because you should be always worried of the risk to declare something into the global object in js.
I am not sure if it is here or if my suggestion would break the call to this function below

Reply:
Thanks for the clarification! I read in https://dmitripavlutin.com/6-ways-to-declare-javascript-functions/ that using `function is_opton_set(option)` declares a variable in the current scope, so I'm still not sure what the difference is.

Scope is under require('web_m2x_options.web_m2x_options'), not global so I don't think that's an issue.

Reply:
So indeed probably no differences, this https://softwareengineering.stackexchange.com/questions/160732/function-declaration-as-var-instead-of-function looks specific to our discussion

Reply:
Thanks for finding that!